client/src/modules/indicators/indicator.ctrl.js

```diff
@@ -0,0 +1,46 @@
+angular.module('bhima.controllers')
```

Comment:
Could you remove all these client code and keep only services which provide communication to the API at the server